packages/core/scripts/umip-15/2_VoteSimulate.js

```diff
@@ -34,7 +35,7 @@ async function runExport() {
    ***********************************/
 
   console.log("0. SETUP PHASE");
-  const voting = await Voting.deployed();
```

Comment:
this was not added to the umip upgrade scripts clearly but is added to the vote related ones.